.travis.yml

```diff
@@ -5,6 +5,7 @@ env:
   - CMD="sass:compile"
   - CMD="lint:sass"
   - CMD="lint:js"
+  - CMD="test:karma"
```

Comment:
I'm fine with phantomjs and I think it does make sense to separate that work into a diff pr.